ext/handlers_http.h

```diff
@@ -136,12 +137,13 @@ static inline void ddtrace_inject_distributed_headers_config(zend_array *array,
     bool send_b3single = zend_hash_str_exists(inject, ZEND_STRL("b3 single header"));
 
     zend_long sampling_priority = ddtrace_fetch_priority_sampling_from_root();
-    if (get_DD_EXPERIMENTAL_APPSEC_STANDALONE_ENABLED() && DDTRACE_G(asm_event_emitted) == true) {
+    if (!get_DD_APM_TRACING_ENABLED() && ddtrace_asm_event_emitted()) {
         sampling_priority = PRIORITY_SAMPLING_USER_KEEP;
     }
 
-    if (get_DD_EXPERIMENTAL_APPSEC_STANDALONE_ENABLED() && DDTRACE_G(asm_event_emitted) == false &&
-        ddtrace_propagated_tags_get_tag(DD_TAG_P_APPSEC) == NULL) {
+    ddtrace_root_span_data *root_span = DDTRACE_G(active_stack) ? DDTRACE_G(active_stack)->root_span : NULL;
+    zend_array *meta = root_span ? ddtrace_property_array(&root_span->property_meta) : NULL;
```

Comment:
```suggestion
    zend_array *meta = root_span ? ddtrace_property_array(&root_span->property_meta) : &DDTRACE_G(root_span_tags_preset);
```
Otherwise propagation does not work when no trace was created yet.

Reply:
👏 thanks